src/annotator/config/index.js

```diff
@@ -23,6 +23,7 @@ function configFrom(window_) {
     enableExperimentalNewNoteButton: settings.hostPageSetting(
       'enableExperimentalNewNoteButton'
     ),
+    group: settings.group,
```

Comment:
Once we've done a bit more testing and we're happy with this config setting, we'll need to document it. Would you mind adding an item in the relevant GitHub issue so this doesn't get forgotten.